frontend/src/components/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+const Header = () => {
+  return (
+    <>
+      <header>
+        <NavLink exact to="/">
+          Home
+        </NavLink>
+        <NavLink to="/Wines">Wines</NavLink>
+        <NavLink to="/LogIn">Log In</NavLink>
+        <NavLink to="/SignUp">Sign Up</NavLink>
+      </header>
+    </>
```

Comment:
No need for React fragment.
Also, we can wrap the links with a `nav` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav.
And you can use a list to list the nav links.